sources/api/migration/migrations/archived/v1.2.0/cfsignal/src/main.rs

```diff
@@ -0,0 +1,23 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddSettingsMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added a set of settings for configuring service cfsignal.
+/// Remove the whole `settings.cloudformation` prefix if we downgrade.
+fn run() -> Result<()> {
+    migrate(AddPrefixesMigration(vec![
+        "settings.cloudformation",
+    ]))
```

Comment:
There are a few more prefixes that gets added.

```suggestion
    migrate(AddPrefixesMigration(vec![
        "settings.cloudformation",
        "services.cfsignal",
        "configuration-files.cfsignal-toml",
    ]))
```